changelogs/current.yaml

```diff
@@ -8,6 +8,12 @@ minor_behavior_changes:
 - area: http2
   change: |
     Sets runtime guard ``envoy.reloadable_features.http2_use_oghttp2`` to true by default.
+- area: dfp
+  change: |
+    Setting :ref:`dns_query_timeout
+    <envoy_v3_api_field_extensions.common.dynamic_forward_proxy.v3.DnsCacheConfig.dns_query_timeout>`
+    to 0 will disable the DNS query timeout. This behavior can be disabled by setting
+    ``envoy.restart_features.allow_disabling_dns_query_timeout`` flag to ``false``.
```

Comment:
documented here as a restart feature but I'm failing to see the runtime guard change?